docs/Overview.md

```diff
@@ -0,0 +1,44 @@
+# **ONE**-vscode tutorial
+
+In this tutorial, you will use `ONE` driver to compile a model, and refer to and profile the generated backend model in Visual Studio Code. By using the ONE-vscode extension, you make VS Code into a great ONE IDE.
+
+`ONE` driver is a command tool that generates binary running in the NPU backend, executes it and profiles it. Therefore, NPU backend is required to use ONE driver. The currently available backends are as follows:
+- TRIX
+If you want to use your NPU backend in ONE vscode, this can be implemented by redefining the Backend very simply. See the following guide, [How to define new Backend for ONE-vscode](docs/HowToDefineNewBackend.md), for help.
```

Comment:
Q) Did you intend for the missing link of `How to define new Backend for ONE-vscode`? (I found this part is written by @jyoungyun )

Reply:
@jyoungyun ?

Reply:
If we don't have a plan to create backend document, it's better to remove that part.

Reply:
Or how about adding an empty file for this document for now?

Reply:
- [ ] TBD in `docs/HowToDefineNewBackend.md`